ansible/playbooks/adhoc/uninstall.yml

```diff
@@ -114,7 +114,7 @@
     - name: Remove remaining files
       file: path={{ item }} state=absent
       with_items:
-        - "~{{ ansible_ssh_user }}/.kube"
+        - "{{ ansible_env.HOME }}/.kube"
```

Comment:
What is the difference between ``~/.kube`` and ``{{ ansible_env.HOME }}/.kube`` then?